includes/controls/groups/flex-container.php

```diff
@@ -47,8 +47,14 @@ protected function init_fields() {
 				],
 			],
 			'default' => '',
+			'selectors_dictionary' => [
+				'row' => '--flex-direction: row; --container-widget-width: initial;',
```

Comment:
Add a comment here explaining that this is for handling widgets that get an undefined width in column mode.